### PR TITLE
[ID] Add `Community` Category and Update `osu!dev_Discord_server` and `How_you_can_help!`

### PR DESCRIPTION
**Pull request by ZeroPyrozen**
## Self-check

- [x] The changes are tested against the [contribution checklist](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#self-check)
- [x] *(translations only)* The changes are reviewed on GitHub [by a fluent speaker](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#review)

### CODE REVIEW

File: wiki/osu!dev_Discord_server/id.md [hunk 1]
```diff
@@ -1,22 +1,21 @@
...
   - development
 ---
 
 # Server Discord osu!dev
 
-**osu!dev** merupakan server [Discord](https://discordapp.com) resmi yang digunakan perihal proyek *open source* osu! dan proyek yang dijalankan komunitas osu!. Di sinilah tempat utama yang bisa kamu kunjungi untuk berkontribusi pada pengembangan osu! dan berhubungan dengan [tim inti osu!](/wiki/People/The_Team).
+**osu!dev** merupakan server [Discord](https://discordapp.com) resmi yang digunakan untuk proyek *open source* osu! dan proyek yang dijalankan komunitas osu!. Di sinilah tempat utama yang bisa kamu kunjungi untuk berkontribusi pada pengembangan osu! dan berhubungan langsung dengan [tim inti osu!](/wiki/People/The_Team).
 
-Tautan undangan untuk masuk sudah permanen dan bisa diakses dengan <https://discord.gg/ppy>.
+Tautan undangan untuk masuk ke dalam server sudah permanen dan bisa diakses dengan <https://discord.gg/ppy>.
```
Comment (daud-ahn) [inline wiki/osu!dev_Discord_server/id.md:10]:
```suggestion
Tautan undangan permanen untuk masuk ke dalam server bisa diakses dengan <https://discord.gg/ppy>.
```

**Review by daud-ahn — CHANGES_REQUESTED** [review]

File: wiki/Community/osu!dev_Discord_server/id.md [hunk 2]
```diff
@@ -1,22 +1,21 @@
...
   - development
 ---
 
 # Server Discord osu!dev
 
-**osu!dev** merupakan server [Discord](https://discordapp.com) resmi yang digunakan perihal proyek *open source* osu! dan proyek yang dijalankan komunitas osu!. Di sinilah tempat utama yang bisa kamu kunjungi untuk berkontribusi pada pengembangan osu! dan berhubungan dengan [tim inti osu!](/wiki/People/The_Team).
+**osu!dev** merupakan server [Discord](https://discordapp.com) resmi yang digunakan untuk proyek *open source* osu! dan proyek yang dijalankan komunitas osu!. Di sinilah tempat utama yang bisa kamu kunjungi untuk berkontribusi pada pengembangan osu! dan berhubungan langsung dengan [tim inti osu!](/wiki/People/The_Team).
 
-Tautan undangan untuk masuk sudah permanen dan bisa diakses dengan <https://discord.gg/ppy>.
+Tautan undangan permanen untuk masuk ke dalam server dan bisa diakses dengan <https://discord.gg/ppy>.
```
Comment (daud-ahn) [inline wiki/Community/osu!dev_Discord_server/id.md:10]:
```suggestion
Tautan undangan permanen untuk masuk ke dalam server bisa diakses dengan <https://discord.gg/ppy>.
```

File: wiki/Community/How_you_can_help!/id.md [hunk 3]
```diff
@@ -1,74 +1,74 @@
...
 
 Untuk menjaga keamanan hub komunitas resmi, osu! memiliki tim moderasi khusus yang memantau forum dan obrolan online sepanjang waktu.
 
-Untungnya, masih mudah bagi Kamu untuk membantu menjaga semuanya tetap aman. Berikut ini adalah sebagian kecil dari banyak cara yang dapat Kamu lakukan untuk membantu:
+Untungnya, masih mudah bagi kamu untuk membantu menjaga semuanya tetap aman. Berikut ini adalah sebagian kecil dari banyak cara yang dapat kamu lakukan untuk membantu:
 
-- Ikuti kanal favorit Kamu di [obrolan osu! dalam gim](/wiki/Chat_Console), atau di forum.
-- Jika Kamu melihat suatu peristiwa yang tidak pantas atau mengganggu, gunakan perintah `!report` dalam obrolan dalam osu! untuk memberi tahu moderator.
+- Ikuti *channel* favorit kamu di [obrolan osu! dalam permainan](/wiki/Chat_Console) atau di forum.
+- Jika kamu melihat suatu peristiwa yang tidak pantas atau mengganggu, gunakan perintah `!report` dalam obrolan dalam osu! untuk memberi tahu moderator.
```
Comment (Walavouchey) [inline wiki/Community/How_you_can_help!/id.md:20]: link to `/wiki/Reporting_Bad_Behaviour`

image

**Review by Walavouchey — CHANGES_REQUESTED** [review]

File: wiki/Community/How_you_can_help!/id.md (hunk 3 above)
Comment (ZeroPyrozen) [reply wiki/Community/How_you_can_help!/id.md]: Thank you, I also noticed there are terms that need to be linked

File: wiki/Community/id.md [hunk 4]
```diff
@@ -0,0 +1,33 @@
+# Komunitas
+
+Berikut ini merupakan daftar artikel yang membahas seputar cara komunikasi yang terjadi di komunitas osu!. Jika kamu mencari cara berkontribusi tetapi belum tahu harus mulai dari mana, kamu dapat membaca artikel [Bagaimana kamu dapat membantu!](/wiki/Community/How_you_can_help!) untuk mengetahui cara berkontribusi.
+
+## Tempat
+
+<!-- TODO: add forums -->
+
```
Comment (Walavouchey) [inline wiki/Community/id.md:8]: small change in https://github.com/ppy/osu-wiki/pull/6523

```suggestion
- [Forum](/wiki/Community/Forum)
```

**Review by Walavouchey — CHANGES_REQUESTED** [review]
can get this in if everything looks ok

File: wiki/Community/id.md (hunk 4 above)
Comment (ZeroPyrozen) [reply wiki/Community/id.md]: Sure

**Review by Walavouchey — APPROVED** [review]

**Pull request merged**